### PR TITLE
Add a seed for a project with a submitted ROP

### PR DESCRIPTION
**Pull request by lennym**
To test ROP unsubmission/correction

### CODE REVIEW

File: seeds/data/rops.js [hunk 1]
```diff
@@ -33,5 +33,10 @@ module.exports = [
     projectId: '6ae2295c-dd09-4c30-bd19-c67dd7f24cea',
     year: '2021',
     status: 'draft'
+  },
+  {
+    projectId: '0c01cd7d-394b-45bb-99fd-ae31aac6943d',
+    year: '2021',
+    status: 'submitted'
```
Comment (wheelsandcogs) [inline seeds/data/rops.js:40]: I'm sure I previously added a `submittedDate` to all the submitted rops seeds, but they're not visible here? 🤔

File: seeds/tables/rops.js [hunk 2]
```diff
@@ -1,6 +1,30 @@
...
+  scheduleTwoDetails: null,
+  species: JSON.stringify({ precoded: ['mice'], otherSpecies: [] })
+};
+
 module.exports = {
-  populate: knex => knex('rops').insert(rops),
+  populate: knex => knex('rops').insert(
+    rops.map(rop => ({
+      ...defaults,
+      submittedDate: rop.status === 'submitted' ? (new Date()).toISOString() : null,
```
Comment (wheelsandcogs) [inline seeds/tables/rops.js:25]:
```suggestion
      submittedDate: rop.status === 'submitted' ? rop.submittedDate || (new Date()).toISOString() : null,
```

Comment (lennym) [reply seeds/tables/rops.js:25]: That's a no-op since the submitted date is inherited from the `...rop` on the next line.

Comment (wheelsandcogs) [reply seeds/tables/rops.js:25]: doh, saw the `...defaults` and completely missed the `...rop`

**Pull request merged**